adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -294,13 +297,16 @@ private AuthenticationResult acquireTokenSilentFlow(final AuthenticationRequest
     }
 
     /**
-     * Try acquire token silent locally.
-     */
+    * Try acquire token silent locally.
+    */
     private AuthenticationResult tryAcquireTokenSilentLocally(final AuthenticationRequest authenticationRequest)
             throws AuthenticationException {
         Logger.v(TAG, "Try to silently get token from local cache.");
         final AcquireTokenSilentHandler acquireTokenSilentHandler = new AcquireTokenSilentHandler(mContext,
                 authenticationRequest, mTokenCacheAccessor);
+        if(mAuthContext.getExtendedLifetimeEnabled()) {
```

Comment:
>f(mAuthContext.getExtendedLifetimeEnabled() [](start = 9, length = 43)

you never call setOutageModeIsOn without this check, why not move the check inside the setOutageModeIsOn, and maybe rename the function to setOutageModeForExtendedLifetime or something else that makes sense